packages/babel-parser/src/tokenizer/types.ts

```diff
@@ -139,12 +139,17 @@ export type InternalTokenTypes = typeof tt;
 export const tt = {
   // Punctuation token types.
   bracketL: createToken("[", { beforeExpr, startsExpr }),
+  // TODO: Remove this in Babel 8
```

Comment:
These are all not user-exposed, right?

Reply:
Unfortunately it is exposed via the `tokens` parser option. And Babel eslint parser depends on token labels so that it can transform Babel tokens to espree tokens. Although espree does not support R&T, we do have to convert it to a punctuator token type.